<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordGeneratorDialogPage1</class>
 <widget class="QWidget" name="PasswordGeneratorDialogPage1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>326</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>To generate password please define it's length and character set. </string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>305</width>
       <height>70</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Length: </string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Character set:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QWidget" name="widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="smallCharactersBox">
          <property name="text">
           <string>Small characters</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="bigCharactersBox">
          <property name="text">
           <string>Big characters</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="digitsBox">
          <property name="text">
           <string>Digits</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="specialCharactersBox">
          <property name="text">
           <string>Special characters</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="lengthBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>30</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lengthBox</tabstop>
  <tabstop>smallCharactersBox</tabstop>
  <tabstop>bigCharactersBox</tabstop>
  <tabstop>digitsBox</tabstop>
  <tabstop>specialCharactersBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
